src/DataCollector/StatsdDataCollector.php

```diff
@@ -18,12 +18,20 @@ class StatsdDataCollector extends DataCollector
      * Construct the data collector
      */
     public function __construct()
+    {
+        $this->reset();
+    }
+
+    /**
+     * Reset the data collector to initial state
+     */
+    public function reset()
     {
         $this->statsdClients      = [];
         $this->data['clients']    = [];
         $this->data['operations'] = 0;
```

Comment:
could you please write the `reset` function content this way:
```php
$this->statsdClients = [];
$this->data = [
    'clients' => [],
    'operations' => 0,
];
```

as re-instanciating the whole `data` property is how we already did the `reset()` stuff in our other projects ( such as [AmqpBundle](https://github.com/M6Web/AmqpBundle/blob/master/src/AmqpBundle/Amqp/DataCollector.php#L93-L100) or [CassandraBundle](https://github.com/M6Web/CassandraBundle/blob/master/src/DataCollector/CassandraDataCollector.php#L44-L49) )